docs/site/connect.md

```diff
@@ -61,6 +63,19 @@ All this said, I still recommend you challenge the conclusion that you can't use
 !!! Note
     There is a Calva command for copying the Jack-in command line to the clipboard.
 
+## Monorepos / multiple Clojure projects in one workspace
+
+If the workspace is a monorepo, Polylith repo or just a repository with more than one Clojure project, Calva will start the connect sequence with prompting for which project to start/connect to.
+
+![The project roots menu](images/calva-monorepo-project-roots-menu.png)
+
+When searching for project roots in your workspace, Calva will glob for all files matching `project.clj`, `deps.edn`, or `shadow-cljs.edn`. This is done using VS Code's workspace search engine, and is very efficient. However, in a large monorepo, it is still a sufficient task. In order to not waste resources Calva will exclude any directories in the setting `calva.projectRootsSearchExclude`. 
```

Comment:
Did you mean to say "still a substantial task" here? It seems, judging by the context, that you mean to point out that it even though the search engine is very efficient, a large monorepo might still take a noticeable amount of time.

Reply:
Haha, yes. Thanks!